src/models/methods/ledger.ts

```diff
@@ -0,0 +1,58 @@
+import { AccountRoot, Amendments, Check, DepositPreauth, DirectoryNode, Escrow, FeeSettings, LedgerHashes, NegativeUNL, Offer, PayChannel, RippleState, SignerList, Ticket } from "../ledger";
+import { BaseRequest, BaseResponse } from './base_method'
+
+export interface LedgerRequest extends BaseRequest {
+    command: "ledger"
+    ledger_hash?: string,
+    ledger_index?: string | number
+    full?: boolean
+    accounts?: boolean
+    transactions?: boolean
+    expand?: boolean
+    owner_funds?: boolean
+    binary?: boolean
+    queue?: boolean
+}
+
+interface Ledger {
+    account_hash: string
+    accountState?: (AccountRoot | Amendments | Check | DepositPreauth | DirectoryNode
+        | Escrow | FeeSettings | LedgerHashes | NegativeUNL | Offer
+        | PayChannel | RippleState | SignerList | Ticket)[]
```

Comment:
this should be defined as a type so that you can make an array of that type, for legibility.

Reply:
Isn't this just `LedgerEntry[]`?